react/src/components/ShellScriptEditModal.tsx

```diff
@@ -0,0 +1,321 @@
+                        content: t('dialog.ask.DoYouWantToResetChanges'),
+                        onOk: () => {
+                          if (shellInfo === 'bootstrap') {
+                            setScript('');
+                          }
+                          if (shellInfo === 'userconfig') {
+                            setScript('');
+                          }
+                        },
+                      });
+                    },
+                    danger: true,
+                  },
+                ],
+              }}
+              onClick={() => {
+                app.modal.confirm({
+                  title: t('dialog.title.LetsDouble-Check'),
+                  content: t('dialog.ask.DoYouWantToDeleteSomething', {
+                    name:
+                      shellInfo === 'bootstrap'
+                        ? 'Bootstrap script'
+                        : `${rcfileNames}`,
+                  }),
+                  onOk: deleteScript,
+                });
+              }}
+            >
+              <DeleteOutlined />
+            </Dropdown.Button>
+          </Flex>
+          <Flex gap={'sm'}>
+            <Button
+              key="cancel"
+              onClick={() => onRequestClose()}
+              style={{ width: 'fit-content' }}
+            >
+              {t('button.Cancel')}
+            </Button>
+            <Dropdown.Button
+              key="submit"
+              type="primary"
+              onClick={() => {
+                saveScript();
+              }}
+              style={{ width: 'fit-content' }}
+              menu={{
+                items: [
+                  {
+                    key: 'save',
+                    label: t('button.SaveWithoutClose'),
+                    onClick: () => saveScript({ closeAfter: false }),
+                  },
+                ],
+              }}
+            >
+              {t('button.SaveAndClose')}
+            </Dropdown.Button>
+          </Flex>
+        </Flex>,
+      ]}
+      destroyOnClose
+      {...modalProps}
+    >
+      <Flex direction="column" align="stretch" gap={'sm'}>
+        <Typography.Text>
+          {t('usersettings.BootstrapScriptDescription')}
+        </Typography.Text>
+        {shellInfo === 'userconfig' && (
+          <Select
+            onClear={() => setRcfileNames('.bashrc')}
+            defaultValue={'.bashrc'}
+            onChange={(value) => {
+              const selectedScript = _.find(userConfigScript, { path: value });
+              setScript(selectedScript?.data || '');
+              setRcfileNames(value);
+            }}
+            options={[
+              { value: '.bashrc', label: '.bashrc' },
+              { value: '.zshrc', label: '.zshrc' },
+              { value: '.tmux.conf.local', label: '.tmux.conf.local' },
+              { value: '.vimrc', label: '.vimrc' },
+              { value: '.Renviron', label: '.Renviron' },
+            ]}
```

Comment:
You don't need to set Label when value and label are same.
```suggestion
              { value: '.bashrc' },
              { value: '.zshrc' },
              { value: '.tmux.conf.local' },
              { value: '.vimrc' },
              { value: '.Renviron' },
```